pkg/policy/bucket-policy.go

```diff
@@ -620,6 +620,33 @@ func GetPolicies(statements []Statement, bucketName string) map[string]BucketPol
 	return policyRules
 }
 
+// GetPrefixPolicies - returns a map of policies of given bucket name, prefix in given statements.
+func GetPrefixPolicies(statements []Statement, bucketName, prefix string) map[string]BucketPolicy {
```

Comment:
actually a better function name for go style is `GetPoliciesWithPrefix` helps in editor auto completion and also conveys the purpose.

Reply:
done